docs/modules/ROOT/pages/how-tos/testing/integration/rerunning.adoc

```diff
@@ -1 +1,57 @@
-= Rerunning an integration test
\ No newline at end of file
+= Retriggering Integration Tests
+
+Integration test scenarios for a given snapshot can be re-triggered by adding a label to the snapshot.
+
+.Prerequisites
+- You have a snapshot that has completed all its initial tests. All initial tests must be finished before you trigger subsequent tests.
+
+- The snapshot for which tests will be retriggered has completed its initial tests.
+
+- You have CLI access to the specific OpenShift cluster. For information on obtaining CLI access, refer Getting started.
+
+- You have an up-to-date kubectl binary.  Alternatively, the `oc` binary is also compatible.
+
+.Procedure
+
+. Identify the IntegrationTestScenario that needs to be rerun.  For a given snapshot, you can only rerun one IntegrationTestScenario.
```

Comment:
@dirgim @14rcole , do we need to provide assistance here to instruct users in ways to identify the desired Snapshot?

Reply:
The related Snapshot can be found in the pipelineRun details page in the UI, e.g. 
![image](https://github.com/konflux-ci/docs/assets/34320458/187bb3db-4b81-469a-a331-e6f00aa7eb2d)